web/client/epics/wfsdownload.js

```diff
@@ -14,27 +14,24 @@ const DOWNLOAD_FORMATS_LOOKUP = {
 	"GML2": "GML2",
 	"application/vnd.google-earth.kml+xml": "KML",
 	"OGR-CSV": "OGR-CSV",
-	"OGR-FileGDB": "OGR-FileGDB",
-	"OGR-GPKG": "OGR-GPKG",
+	"OGR-FileGDB": "OGR-GeoDatabase",
+	"OGR-GPKG": "OGR-OGR-GeoPackage",
 	"OGR-KML": "OGR-KML",
 	"OGR-MIF": "OGR-MIF",
 	"OGR-TAB": "OGR-TAB",
 	"SHAPE-ZIP": "Shapefile",
 	"gml32": "GML3.2",
 	"application/json": "GeoJSON",
 	"csv": "CSV",
-	"application/x-gpkg": "application/x-gpkg",
-	"geopackage": "geopackage",
-	"geopkg": "geopkg",
-	"gpkg": "gpkg"
+	"application/x-gpkg": "GeoPackage"
 };
 
-const hasOutputFormat = (data) => {
+const hasOutputFormat = (data, formatFilter) => {
     const operation = get(data, "WFS_Capabilities.OperationsMetadata.Operation");
     const getFeature = find(operation, function(o) { return o.name === 'GetFeature'; });
     const parameter = get(getFeature, "Parameter");
     const outputFormatValue = find(parameter, function(o) { return o.name === 'outputFormat'; }).Value;
-    const pickedObj = pick(DOWNLOAD_FORMATS_LOOKUP, outputFormatValue);
+    const pickedObj = pick(!isEmpty(formatFilter) ? formatFilter : DOWNLOAD_FORMATS_LOOKUP, outputFormatValue);
```

Comment:
So formatFilter is an alternative to DOWNLOAD_FORMATS_LOOKUP ?

Reply:
yes, it's ok?

Reply:
acceptable